misc-scripts/xref_mapping/XrefParser/Database.pm

```diff
@@ -23,22 +23,26 @@ use strict;
 use warnings;
 use Carp;
 use DBI;
-
+use Bio::EnsEMBL::DBSQL::DBConnection;
```

Comment:
The changes in XrefParser::Database that ENSCORESW-2831 wanted are done, except with composition instead of inheritance.

Is there a separate ticket to use disconnect_if_idle where appropriate? Or it's part of ENSCORESW-2831?

Reply:
It is part of the same ticket, as it was the main reason for the change